ghost/core/core/server/services/slack-notifications/service.js

```diff
@@ -0,0 +1,29 @@
+const DomainEvents = require('@tryghost/domain-events');
+const labs = require('../../../shared/labs');
+
+class SlackNotificationsServiceWrapper {
+    /** @type {import('@tryghost/slack-notifications/lib/slack-notifications')} */
+    #api;
+
+    init() {
+        if (this.#api) {
+            // Prevent creating duplicate DomainEvents subscribers
+            return;
+        }
+
+        const SlackNotificationsService = require('@tryghost/slack-notifications');
+        const settingsCache = require('../../../shared/settings-cache');
+        const config = require('../../../shared/config');
+
+        this.#api = new SlackNotificationsService({
+            DomainEvents,
+            labs,
+            settingsCache,
```

Comment:
I can't see that this is used anywhere 🤔

Reply:
lol, my bad. Had this already changed and updated, but not pushed 🤦